main.py

```diff
@@ -179,24 +179,18 @@ def chooser_callback(self,uri_list):
         '''
         try:
             ss = SharedStorage()
-            for uri in uri_list: # only one list
-                
-                # copy to private cache
-                path = ss.copy_from_shared(uri)
-
-                #read content from cache
-                try:
-                    with open(path,'r',encoding = 'utf-8') as fread:
-                        reading = fread.read()
-
-                    #Clock is needed otherwise throw thread error
-                    Clock.schedule_once(lambda dt: self.populate_label(reading))
+            # copy to private cache
+            path = ss.copy_from_shared(uri[0])
+            
+            #read content from cache
+            with open(path,'r',encoding = 'utf-8') as fread:
+                reading = fread.read()
 
-                except Exception as e:
-                    toast(f"1.cannot open file path : {str(path)}  because : {str(e)}")
+            #Clock is needed otherwise throw thread error
+            Clock.schedule_once(lambda dt: self.populate_label(reading))
```

Comment:
@javixav check the review below and remove the use of `Clock` and call `self.populate_label(reading)` directly